.travis.yml

```diff
@@ -1,22 +1,53 @@
+env:
+  global:
+    - REPO_DIR=.
+    - BUILD_COMMIT=$TRAVIS_COMMIT
+    - PLAT=x86_64
+
+install: 
+  - git clone https://github.com/matthew-brett/multibuild
+  - . multibuild/common_utils.sh
```

Comment:
is this sourced by travis_steps.sh?

Reply:
I am not 100% sure I know how visibility and sourcing works in bash (dark sorcery it is to me, ha-ha), but without sourcing `common_utils.sh` the `build_wheel` command fails